trojsten/submit/views.py

```diff
@@ -300,6 +303,21 @@ def poll_submit_info(request, submit_id):
     }), content_type='application/json; charset=utf-8')
 
 
+def send_notification_email(submit):
+    send_mail(
+        _('[Trojstenweb] User submission detected'),
+        _('{name} submitted solution to task {task}\n\nSubmit link: {submit_link}\n\nThis is an automated'
+          ' response, do not reply').format(
+            name=submit.user.get_full_name(),
+            task=str(submit.task),
+            submit_link=Site.objects.get_current().domain + reverse('admin:old_submit_submit_change',
```

Comment:
Toto je už fajn, ale ako som si všimol, v submit adminovi nie je priamo možnosť opraviť popis.
Na to musí ísť vedúci cez úlohy-> nájde svoju úlohu -> klikne na nej na review -> nájde používateľa -> klikne že chce opraviť jeho cez view `admin/contests/task/<task_id>/review/edit/<submit_id>/`

Jeden návrh ako toto zlepšiť je dať v submit adminovi priamo link na opravovanie vpravo hore (tak ako je to spravené [tu](https://github.com/trojsten/web/blob/master/trojsten/contests/admin.py#L150) a [tu](https://github.com/trojsten/web/blob/master/trojsten/reviews/templates/admin/task_review.html)).
 
Problém môže byť to, že vedúci chce opravovať posledný submit k úlohe.

Možno lepšie riešenie by bolo do mailu vložiť ešte jeden link na `admin/contests/task/<task_id>/review/` s popisom že tu môže opraviť riešenie - a možno vložiť tento link skôr ako link na submit, keďže opravovanie je pre vedúceho asi dôležitejšie ako fieldy submitu.

Link na opravovanie je ale relevantý len pre description submit.